meson.build

```diff
@@ -3,7 +3,7 @@
 project(
   'bluechi',
   'c',
-  version: '0.6.0',
+  version: run_command(meson.current_source_dir() / 'build-scripts' / 'version.sh', ' short', check: true).stdout().strip(),
```

Comment:
I'd use `join_paths` function here instead of string concatenation.

Also maybe i would be good to store path to `version.sh` to some variable and then use it for both invocations

Reply:
Good point! Will change that. 
And yes, would be great if I could reuse it, but I am afraid this is not possible (as `project` needs to be the first statement in meson). Maybe we could outsource the script path to the meson options? I'll give it a try.

Reply:
I am currently using the short version in project (e.g. `0.6.0-1`). Could we also use the long version? In that case, I can exchange the second `run_command` with a simple `meson.project_version()`. What do you think? @mwperina

Reply:
No, I'd rather not mix version and release. So let's leave as is